backend/code_review_backend/issues/management/commands/cleanup_issues.py

```diff
@@ -29,24 +29,38 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         clean_until = timezone.now() - timedelta(days=options["nb_days"])
+
+        # For issue and issuelink we can use _raw_delete since it's faster and doesn't
+        # give OOM since there are no CASCADE directives in the model since django
+        # doesn't need to query the model and compute the dependencies.
         rev_to_delete = Revision.objects.filter(
             base_repository__slug__in=["autoland", "mozilla-central"],
             head_repository__slug__in=["autoland", "mozilla-central"],
             created__lte=clean_until,
         )
+        count_rev = rev_to_delete.count()
 
-        count = rev_to_delete.count()
-        if not count:
+        if not count_rev:
             logger.info("Didn't find any old revision to delete.")
             return
 
-        # Delete revisions, with their related IssueLink and Diff
+        issuelinks_to_delete = IssueLink.objects.filter(revision_id__in=rev_to_delete)
+        issues_to_delete = Issue.objects.filter(
+            id__in=issuelinks_to_delete.values("issue_id")
```

Comment:
```suggestion
            issue_links__revision_id__in=rev_to_delete
```
Saves a request and potentially avoid an OOM by storing all the IDs from the IssueLink table.